go/packages/api/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "api",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "application",
+  "sourceRoot": "apps/api",
+  "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "packages/api",
+        "commands": ["go build -o ../../dist/packages/api ./main.go"]
```

Comment:
I think you should be able to use just `command` here, instead of `commands` and an array, since you just have one. Make sure you try first though.

this applies on all the others as well

Reply:
updated